decidim-core/config/locales/en.yml

```diff
@@ -1168,6 +1168,10 @@ en:
           one: "%{count} result"
           other: "%{count} results"
         view_all: View all (%{count})
+    security:
+      selfxxs_warning:
+        description: This browser feature is meant for developers and you should not paste anything here if you were asked to do so. Pasting content in this window can compromise your privacy and give hackers access to your account.
```

Comment:
I personally prefer using the term "cracker" instead of "hacker", but I don't know if our visitors/participants would understand it :no_mouth:

Reply:
Are you sure @andreslucena ?

https://en.wikipedia.org/wiki/Cracker_(term)

Reply:
Yes, but it seems that not even Wikipedia is on my side: https://en.wikipedia.org/wiki/Security_hacker: 

> Longstanding controversy surrounds the meaning of the term "hacker". In this controversy, computer programmers reclaim the term hacker, arguing that it refers simply to someone with an advanced understanding of computers and computer networks and that cracker is the more appropriate term for those who break into computers, whether computer criminals (black hats) or computer security experts (white hats). A 2014 article noted that "... the black-hat meaning still prevails among the general public".

More information: http://www.stallman.org/articles/on-hacking.html

But as I said, on this case I'd prefer something that everybody understands.

Reply:
I would also go with the term that doesn't require the participant being a computer security expert. I am aware of these two terms in the computer security context but honestly I've never heard anyone even in the industry using the word "cracker" in common spoken language. Especially not in Finnish, unless the person is a computer security expert.

I think in the general language, the word hacker has a negative connotation, people know what it is and there is no side meaning connected with that term (at least that I know of).